### ISSUE TITLE
Markus - Content/Extension mismatch

### ISSUE THREAD

**Issue by zengzix2** [dilevin/computer-graphics-raster-images#73]
My program compiles and runs and produces the correct outputs on teach.cs.
But when I upload the .cpp files to MarkUs, I get a warning saying 
"The contents of the uploaded file do not match the file extension .cpp.".

Should I ignore this warning?

**Comment by Zhecheng-Wang** [comment]
Hmm, not sure why this is happening, but as long as it is a valid .cpp file it is fine. Can you double check if you are uploading a .cpp file?

**Comment by Zhecheng-Wang** [comment]
Can you try to submit? I will check on the grader side.

**Comment by zengzix2** [comment]
Yes, the files are successfully submitted. There's just this weird warning.
Besides, even if I try to upload the main.cpp file from the handout (though not submitted), the same warning also shows up.

**Comment by Zhecheng-Wang** [comment]
Oh, I am a bit surprised you can upload the main.cpp. There should be a restriction on the file name (and extension).

**Comment by zengzix2** [comment]
No, I can't. But MarkUs somehow still verifies the contents? and shows the same warning.

**Comment by Zhecheng-Wang** [comment]
Right, I think markus doesn't collect HW before ddl, so I can't see it right now. But if it is successfully uploaded I wouldn't worry about it. Thanks for bringing this up though!

**Comment by jennicao** [comment]
Just commenting that I got the same "The contents of the uploaded file do not match the file extension .cpp." warning when uploading the .cpp files to MarkUs. I'm assuming we can ignore it based on this discussion.

**Comment by Zhecheng-Wang** [comment]
I checked your submission and it looks fine. For everyone running into the same issue, please ignore this warning. I think as long as MarkUs allows you to submit it's fine.
